src/net/sourceforge/kolmafia/maximizer/Evaluator.java

```diff
@@ -1843,7 +1831,6 @@ void enumerateEquipment(EquipScope equipScope, int maxPrice, PriceLevel priceLev
       MaximizerSpeculation synergySpec = new MaximizerSpeculation();
       MaximizerSpeculation compareSpec = new MaximizerSpeculation();
 
-      Slot newSlot1 = slot1;
       int compareItemNo =
```

Comment:
while technically newSlot1 is redundant, I feel like it improves clarity by symmetry with the lower methods to have it present.

Reply:
Restored